Update nhood-examples info (#75)
nhood-org/nhood-docs: README.md

## Use-case examples

> There is a couple of examples implemented on an engine library level as ready-to-run unit tests. You can find those in `nhood-examples` module.

Reviewer: ```suggestion
There is a couple of examples implemented as ready-to-run unit tests. You can find those in [nhood-examples](https://github.com/nhood-org/nhood-examples) repository.
```
